UnityProject/Assets/Scripts/UI/Cargo/GUI_CargoTab.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GUI_CargoTab : NetPage
```

Comment:
would be better if it was called GUI_CargoPage, because it's confusing (as Tab is assumed to be the entire machine UI)